client/server/server.js

```diff
@@ -81,9 +81,9 @@ function evaluateKaoFile(ctx, diagnostics) {
     let visitor = new YmlToBdlVisitorImpl_1.YmlToBdlVisitorImpl(diagnostics, completionItems);
     visitor.visit(ctx);
 }
-connection.onDidChangeWatchedFiles((_change) => {
+connection.onDidChangeWatchedFiles(_change => {
     // Monitored files have change in VSCode
-    console.log('Yseop.vscode-yseopml − We received a file change event');
+    console.log("Yseop.vscode-yseopml − We received a file change event");
```

Comment:
Sounds like a company's motto, with this “foo, blah-blah” structure xd
I suppose this is for debugging, but aren't there _a lot_ of occurrences of these events?
But well, it's kinda unrelated to this PR. Just curious about stuff.